tdrs-backend/tdpservice/email/helpers/account_access_requests.py

```diff
@@ -0,0 +1,26 @@
+"""Helper methods for sending emails to users about their account deactivation."""
+from tdpservice.email.email_enums import EmailType
+from tdpservice.email.email import log, prepare_email, prepare_recipients, send_email
+
+
+def send_num_access_requests_email(recipient_email, text_message, subject, email_context, logger_context=None):
+    """Send an email to OFA System Admins notifying them how many access requests there are.
+
+    recipient_email can be either a string (single recipient) or a array of strings.
+    """
+    email_path = EmailType.ACCESS_REQUEST_COUNT.value
+    prepare_recipients(recipient_email)
```

Comment:
the return value from `prepare_recipients` isn't captured, do we need the function call?